plugin.video.teliaplay-se/resources/lib/api.py

```diff
@@ -60,6 +40,25 @@ def __init__(self, userdata):
             self.token_data = None
         self.web_utils = WebUtils()
 
+    @property
+    def graphql_hashes(self):
+        request = {
+            "GET": {
+                "scheme": "https",
+                "host": "raw.githubusercontent.com",
```

Comment:
@enen92 Do we mind this being on his github user content space?

Reply:
Yeah :/

Reply:
Would be possible to include if I made it optional and with a big disclaimer?

Reply:
Does it really changes that frequently that can't be handled via repo submissions/updates?

Reply:
My impression is that it varies; sometimes they become obsolete after ~2 weeks and sometimes they have worked for more than a month. It also seem to vary depending on the request; i.e., some menu entries stops working more frequently than others. I suppose the issue is that they change both frequently and unpredictably, and having a separate repository allows me to respond quicker.

I guess the optimal solution would be to generate the hashes on the fly, or send the full graphql request (if that's even is possible). I've tried figuring out how the browser does it, but not with much luck as my current knowledge of graphql is, let's say, limited... My guess is that they are retrieved/generated through some javascript routine? I would be most grateful if you (or anyone) could provide me with some pointers in this direction. :)

Reply:
Maybe some better option is to look into their hypothetical mobile applications and check how they do it there. I doubt they issue mobile application updates on a 2 week basis

Reply:
Great, will do, thank you! I will keep updating the hashes through PRs then.

Reply:
I you update the PR, I can re-review it.

Reply:
Done; hopefully everything looks alright now.